hypha/apply/activity/templates/messages/email/invoice_status_updated.html

```diff
@@ -2,15 +2,18 @@
 
 {% load i18n %}
 {% block content %}
-{% blocktrans with title=source.title date_from=invoice.date_from date_to=invoice.date_to %}An {{ ORG_SHORT_NAME }} staff member has updated your invoice for {{ title }} for period {{ date_from }} to {{ date_to }}.{% endblocktrans %}
+
+{% blocktrans with title=source.title date_from=invoice.date_from date_to=invoice.date_to %}An {{ ORG_SHORT_NAME }} staff member has updated your invoice for {{ date_from }} for period to {{ date_to }}.{% endblocktrans %}
```

Comment:
I do not think this change is intended.

Reply:
Updated changes as mentioned in the comment.